hedera-node/src/main/java/com/hedera/services/state/expiry/ExpiryRecordsHelper.java

```diff
@@ -120,11 +120,11 @@ private void stream(
         recordStreaming.streamSystemRecord(rso);
     }
 
-    private ExpirableTxnRecord.Builder forTouchedAccount(
-            final EntityNum expiryNum, final Instant consensusTime, final TxnId txnId) {
+    private ExpirableTxnRecord.Builder baseRecordWith(
+            final Instant consensusTime, final TxnId txnId) {
         final var at = RichInstant.fromJava(consensusTime);
         final var receipt = new TxnReceipt();
-        receipt.setAccountId(expiryNum.toEntityId());
+        receipt.setStatus(SUCCESS_LITERAL);
```

Comment:
we don't need to set AccountId , since the `synthTxnBody` has it set ?

Reply:
> we don't need to set AccountId , since the synthTxnBody has it set or the AccountId is only reserved for CryptoCreate ?

Both those points are correct; when HIP-16 was written, we weren't including synthetic `TransactionBody`'s in the record stream. So putting the `AccountID` in the record's receipt was a (confusing) work-around.